samza-core/src/main/java/org/apache/samza/storage/TaskSideInputStorageManager.java

```diff
@@ -211,7 +211,7 @@ private boolean isPersistedStore(String storeName) {
 
   private void validateStoreConfiguration(Map<String, StorageEngine> stores) {
     stores.forEach((storeName, storageEngine) -> {
-      if (storageEngine.getStoreProperties().isLoggedStore()) {
+      if (storageEngine.getStoreProperties().isDurableStore()) {
```

Comment:
This should still be isLoggedStore. Side input stores are durable, this is just checking that they don't have a changelog too (since side input == changelog)